sw.js

```diff
@@ -0,0 +1 @@
+src/platform/web/service-worker.js
```

Comment:
Hope this won't prevent a checkout on windows, but https://github.com/git-for-windows/git/wiki/Symbolic-Links suggests it will just be ignored.